lms/djangoapps/support/views/program_enrollments.py

```diff
@@ -6,18 +6,22 @@
 import csv
 from uuid import UUID
 
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
```

Comment:
you might want to check the import order for the imports introduced here.

Reply:
Updated.